docs/Search.md

```diff
@@ -5,21 +5,161 @@ parent: Features
 ---
 
 {: .note }
-> This only applies to built-in search, other search plugins like may override this syntax.
+> This only applies to the built-in search. Search plugins may override this syntax.
 
-Search query consists of several keywords. Keyword starting with "-" is considered a negative match. Several special keywords are available:
+A search query consists of one or several keywords.
 
-* ``@{date}`` - match by date. For example, @yesterday or @2011-11-03. Please note that due to incomplete implementation, special date keywords like yesterday might not match all articles if user timezone is different from tt-rss internal timezone (UTC).
-* ``pub:{true,false}`` - match only published or unpublished articles
-* ``star:{true, false}`` - same, starred articles
-* ``unread:{true, false}`` - self explanatory (requires trunk as of 05.03.2015)
-* ``note:{true, false, sometext}`` - same, for articles having an attached note or matching the specified text
-* ``label:Somelabel`` - articles that belong to a specified label
-* ``tag:mytag`` - articles which have specified tag
-* ``title:``, ``author:`` - self explanatory
+<a id="text_keyword"></a>A keyword can be a **text keyword**. A text keyword is a single word such as `ocean`, or successive words enclosed in quotes such as `"pacific ocean"`. These keywords are searched using PostgreSQL [Full Text Search](#full_text_search) engine. This engine supports [word stemming](#word_stemming), and [logical operators](#logical_operators).
```

Comment:
* Please go with kebab case (e.g. `text-keyword`) to match what Just the Docs generates for headlines.
* `using PostgreSQL's`